package.json

```diff
@@ -38,6 +38,8 @@
   "devDependencies": {
     "@changesets/cli": "^2.29.5",
     "@eslint-react/eslint-plugin": "^1.52.2",
+    "@html-eslint/eslint-plugin": "^0.42.0",
+    "@html-eslint/parser": "^0.42.0",
```

Comment:
Nit, is installing this independently needed? This is a dependency of `@html-eslint/eslint-plugin` itself.